gobblin-example/src/main/resources/wikipedia-kafka.pull

```diff
@@ -0,0 +1,31 @@
+job.name=PullFromWikipedia
```

Comment:
nit: rename the job, its same as classic Wikipedia example. That will cause conflicts when running in cluster modes such as Yarn